src/server_manager/web_app/app.ts

```diff
@@ -17,11 +17,11 @@ import * as semver from 'semver';
 
 import * as digitalocean_api from '../cloud/digitalocean_api';
 import * as errors from '../infrastructure/errors';
-import * as server from '../model/server';
 import {sleep} from '../infrastructure/sleep';
-import {Account} from '../model/account';
+import * as server from '../model/server';
 
-import {TokenManager} from './digitalocean_oauth';
+import {CloudAccounts} from './cloud_accounts';
+import {DigitalOceanAccount, Status} from './digitalocean_account';
```

Comment:
I'd do something like
```js
import * as do from ./digitalocean;
```

Then use do.Account, do.Status, do.Server.

Reply:
Can we do
```js
import {DigitalOceanAccount as do.Account, Status as do.Status} from './digitalocean_account';
```
?

This would allow the server to be separate if we need to.

Reply:
`do` is a reserved keyword unfortunately, so I used `digitalocean`.